src/Xablu.WebApiClient/BaseClient.cs

```diff
@@ -19,29 +18,7 @@ public BaseClient(IWebApiClient apiClient)
 
         protected async Task<TResult> ExecuteRemoteRequest<TResult>(Func<Task<TResult>> action)
         {
-            TResult result = default(TResult);
-
-            try
-            {
-                result = await Policy
-                    .Handle<WebException>()
-                    .WaitAndRetryAsync
-                    (
-                        retryCount: 5,
-                        sleepDurationProvider: retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt))
-                    )
-                    .ExecuteAsync(action).ConfigureAwait(false);
-            }
-            catch (Exception e)
-            {
-                if (e.GetType().Namespace == "Java.Net")
-                {
-                    throw new HttpExtensions.HttpResponseException(HttpStatusCode.RequestTimeout, e.Message, e.StackTrace);
-                }
-                throw;
-            }
-
-            return result;
+            return await WebApiClientPollyExtensions.PollyDecorator(action, 5, 5);
```

Comment:
Maybe a `sleepDuration` of 5 seconds is a bit long for default? Is it an idea to change this to 1 or 2? And maybe the retry to 3?